rpcs3/Emu/Cell/MFC.cpp

```diff
@@ -298,15 +300,48 @@ void mfc_thread::cpu_task()
 
 				if (completed && spu.ch_tag_upd.compare_and_swap_test(1, 0))
 				{
+					spu.ch_event_stat |= SPU_EVENT_TG;
```

Comment:
This should probably be switched with the line below it, since we use ch_event_stat to trigger interrupt, the jump could happen between these two statements. The interrupt handler could then end up not 'seeing' the ch_tag_stat update then